src/Infrastructure/Resources/config/doctrine/Workflow.orm.yml

```diff
@@ -16,12 +16,27 @@ SprykerSdk\Sdk\Infrastructure\Entity\Workflow:
             type: string
             nullable: false
             unique: false
+        code:
+            type: string
+            nullable: false
+            unique: false
         status:
             type: json
             nullable: false
             unique: false
+    oneToMany:
+        children:
+            targetEntity: Workflow
+            mappedBy: parent
+        transitions:
```

Comment:
cascade delete here. if we remove somehow workflow.

Reply:
deleted also children workflows